### PR TITLE
fix: message approval UI

### PR DESCRIPTION
**Pull request by Schwartz10**
Fix https://github.com/ChainSafe/filsnap/issues/143
Fix https://github.com/ChainSafe/filsnap/issues/144

### CODE REVIEW

File: packages/snap/src/rpc/signMessage.ts [hunk 1]
```diff
@@ -48,12 +49,14 @@ export async function signMessage(
         textAreaContent: messageCreator(
           [
             {message: 'to:', value: message.to},
-            {message: 'value:', value: message.value !== '0' && message.value},
+            {message: 'from:', value: message.from},
```
Comment (BeroBurny) [inline packages/snap/src/rpc/signMessage.ts:52]: now is data show twice
image

If we going with approach to include it in `textAreaContent` it should be be good idea to remove it from `description`

**Review by BeroBurny — COMMENTED** [review]
To address the failing build did you run `build` command from the snap folder before the last commit?

File: packages/snap/src/rpc/signMessage.ts (hunk 1 above)
Comment (BeroBurny) [reply packages/snap/src/rpc/signMessage.ts:52]: yfi value is formatted correctly its my mistake by coping from changes :see_no_evil:

Comment (Schwartz10) [reply packages/snap/src/rpc/signMessage.ts:52]: I don't have a strong opinion on whether or not to remove from the description, but I do like the idea of showing the user _exactly_ the message they're signing (including the `from`) in the `textAreaContent`. I think it's a more secure UI practice to group the message arguments together.

**Comment by Schwartz10** [comment]
> To address the failing build did you run `build` command from the snap folder before the last commit?

@BeroBurny just ran it, but still seems to be failing

**Review by BeroBurny — APPROVED** [review]
:package:

**Pull request merged**